rs/rosetta-api/icrc1/ledger/sm-tests/src/lib.rs

```diff
@@ -2479,11 +2480,9 @@ pub fn icrc1_test_approval_upgrade<T>(
         },
     ];
     let mut initial_balances = vec![];
-    for account in &accounts {
-        initial_balances.push((*account, 10_000_000u64));
-    }
-    for account in &additional_accounts {
-        initial_balances.push((*account, 10_000_000u64));
+    let all_accounts = [accounts.clone(), additional_accounts.clone()].concat();
+    for (index, account) in all_accounts.iter().enumerate() {
+        initial_balances.push((*account, 10_000_000u64 + index as u64));
```

Comment:
Any reason for the change in balances?

Reply:
I only wanted the balances to be different. I guess most are going to be different anyway, as we do some approvals later. Should I remove the index?